resdk/resolwe.py

```diff
@@ -74,8 +74,10 @@ class Resolwe(ResolweUtilsMixin):
     def __init__(self, username=None, password=None, url=None):
         """Initialize attributes."""
         if url is None:
-            # Try to get URL from environmental variable, otherwise fallback to default.
-            url = os.environ.get('RESOLWE_API_HOST', DEFAULT_URL)
+            # Try to get URL from environmental variables, otherwise fallback to default.
```

Comment:
Change this to to: `# Try to get URL from environment variable, otherwise fallback to default URL.`